src/app/notifications/components/SearchFailure/styles.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled, { css, keyframes } from 'styled-components/macro';
+import { PlainButton } from '../../../components/Button';
+import Times from '../../../components/Times';
+import {
+  bookBannerDesktopMiniHeight,
+  bookBannerMobileMiniHeight,
+  toolbarDesktopHeight,
+  toolbarMobileHeight,
+  toolbarMobileSearchWrapperHeight
+} from '../../../content/components/constants';
+import { disablePrint } from '../../../content/components/utils/disablePrint';
+import theme from '../../../theme';
+import { inlineDisplayBreak } from '../../theme';
+import { Header } from '../Card';
+
+export const clearErrorAfter = 3200;
+const fadeOutDuration = 1000;
+
+const bannerBackground = '#F8E8EB';
+const errorBorderColor = '#E297A0';
+const closeIconClor = '#EDBFC5';
+const hoveredCloseIconColor = errorBorderColor;
+
+const fadeOut = keyframes`
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const BannerBodyWrapper = styled.div`
+  width: 100%;
+  margin: 0;
+  height: 0;
+  z-index: ${theme.zIndex.contentNotifications};
+  overflow: visible;
+  position: sticky;
+  top: ${bookBannerDesktopMiniHeight + toolbarDesktopHeight}rem;
+
+  @media (max-width: ${inlineDisplayBreak}) {
+    top: ${bookBannerMobileMiniHeight + toolbarMobileHeight + toolbarMobileSearchWrapperHeight}rem;
+    z-index: calc(${theme.zIndex.searchSidebar} + 1);
```

Comment:
i don't think you want this here isn't the one from line 39 correct on mobile too?